IdentityCore/src/cache/metadata/MSIDAccountMetadataCacheAccessor.m

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import "MSIDAccountMetadataCacheAccessor.h"
+#import "MSIDConfiguration.h"
+#import "MSIDRequestParameters.h"
+#import "MSIDMetadataCache.h"
+#import "MSIDAccountMetadataCacheKey.h"
+#import "MSIDAccountMetadataCacheItem.h"
+
+@implementation MSIDAccountMetadataCacheAccessor
+{
+    MSIDMetadataCache *_metadataCache;
```

Comment:
nit: why don't we use a property instead?

Reply:
I don't see why not, but at the same time, I don't really see a big need here.
For its context, we are using it privately, and I guess it saves couple of lines :p